ptx/sec_deriv_intro.ptx

```diff
@@ -1093,7 +1166,20 @@
           <caption>A graph of the derivative of <m>f(x) = \abs{x}</m></caption>
           <!-- START figures/fig_absolutevalueprime.tex -->
           <image xml:id="img_absolutevalueprime" width="47%">
-            <description></description>
+            <shortdescription>
+              The graph of derivative of function of absolute value of x f(x) = \abs{x}.
```

Comment:
I don't think `\abs{x}` will work with the screen reader. Just use
"The graph of the derivative of the absolute value function"